README.md

```diff
@@ -66,13 +66,13 @@ All additional compatibility is only best effort, or happens to still/already be
 #### Compatibility matrix
 At most, 5 kube-state-metrics and 5 [kubernetes releases](https://github.com/kubernetes/kubernetes/releases) will be recorded below.
```

Comment:
As it states here, we at most show 5, so we need to remove 1.13 from the list.

Reply:
ok, updated it. sorry, missed it.